packages/ui/src/ValueViewComponent/AssetIcon/index.tsx

```diff
@@ -15,7 +15,6 @@ const IconImg = styled.img`
 
 export interface AssetIcon {
   metadata?: Metadata;
-  size?: 'sparse' | 'dense';
```

Comment:
This actually wasn't even being used in this component — it was a leftover that I forgot to remove.